src/status_im2/common/mute_chat_drawer/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.common.mute-chat-drawer.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo2]
+            [react-native.core :as rn]
+            [status-im2.constants :as constants]
+            [utils.re-frame :as rf]
+            [status-im2.common.mute-chat-drawer.style :as style]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+(defn mute-chat-drawer [chat-id accessibility-label]
+  (fn []
```

Comment:
Why the anonymous function?

Reply:
Removed, Added it as I thought I might need to separate logic from render .. But will do it when I need to